sql/updates/world/3.3.5/2017_12_23_99_world.sql

```diff
@@ -0,0 +1,338 @@
+DELETE FROM `creature_addon` WHERE `guid`=@NPC;
+INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES (@NPC,@PATH,0,0,1,0, '');
+DELETE FROM `waypoint_data` WHERE `id`=@PATH;
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(@PATH,1,-696.4537,2672.248,94.02368,0,0,0,0,100,0), -- 15:22:22
+(@PATH,2,-715.4374,2665.521,94.87578,0,0,0,0,100,0), -- 15:22:23
+(@PATH,3,-691.6254,2645.292,93.06143,0,0,0,0,100,0), -- 15:22:25
+(@PATH,4,-687.4556,2661.372,93.48894,0,0,0,0,100,0), -- 15:22:26
+(@PATH,5,-696.2938,2672.279,94.02504,0,0,0,0,100,0), -- 15:22:27
+(@PATH,6,-715.679,2654.232,94.33944,0,0,0,0,100,0), -- 15:22:28
+(@PATH,7,-691.636,2645.265,93.06179,0,0,0,0,100,0), -- 15:22:29
+(@PATH,8,-687.5698,2661.478,93.46684,0,0,0,0,100,0), -- 15:22:31
+(@PATH,9,-706.7507,2672.399,94.58054,0,0,0,0,100,0), -- 15:22:32
+(@PATH,10,-715.6765,2654.364,94.39523,0,0,0,0,100,0), -- 15:22:33
+(@PATH,11,-691.6421,2645.341,93.04417,0,0,0,0,100,0), -- 15:22:34
+(@PATH,12,-687.571,2661.364,93.44951,0,0,0,0,100,0); -- 15:22:35
+-- 0x203CC84240124A400000440006351C34 .go -696.4537 2672.248 94.02368
+
+-- Pathing for Infernal Rain (Hellfire) Entry: 18729 'TDB FORMAT' 
+SET @NPC := @CGUID+9;
+SET @PATH := @NPC * 10;
+UPDATE `creature` SET `spawndist`=0,`MovementType`=2,`position_x`=-736.5909,`position_y`=2643.926,`position_z`=99.57889 WHERE `guid`=@NPC;
+DELETE FROM `creature_addon` WHERE `guid`=@NPC;
+INSERT INTO `creature_addon` (`guid`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES (@NPC,@PATH,0,0,1,0, '');
+DELETE FROM `waypoint_data` WHERE `id`=@PATH;
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES
+(@PATH,1,-736.5909,2643.926,99.57889,0,0,0,0,100,0), -- 15:21:12
+(@PATH,2,-739.5054,2650.205,101.1051,0,0,0,0,100,0), -- 15:21:13
+(@PATH,3,-751.2732,2672.356,100.7091,0,0,0,0,100,0), -- 15:21:14
+(@PATH,4,-739.7455,2691.94,104.7795,0,0,0,0,100,0), -- 15:21:15
+(@PATH,5,-725.373,2704.028,98.12045,0,0,0,0,100,0), -- 15:21:31
+(@PATH,6,-719.1994,2702.7,96.82434,0,0,0,0,100,0), -- 15:21:16
+(@PATH,7,-704.6559,2687.181,95.32121,0,0,0,0,100,0), -- 15:21:18
+(@PATH,8,-725.9246,2683.021,99.72587,0,0,0,0,100,0), -- 15:21:19
+(@PATH,9,-741.8126,2673.67,98.87859,0,0,0,0,100,0), -- 15:21:20
+(@PATH,10,-734.7253,2659.843,97.42217,0,0,0,0,100,0), -- 15:21:21
+(@PATH,11,-720.5742,2666.832,95.61575,0,0,0,0,100,0), -- 15:21:23
+(@PATH,12,-731.9807,2650.673,97.47429,0,0,0,0,100,0), -- 15:21:24
+(@PATH,13,-715.4817,2625.815,91.41167,0,0,0,0,100,0); -- 15:21:25
+-- 0x203CC84240124A400000440007351C34 .go -736.5909 2643.926 99.57889
+
+-- Add text for watch_commander_leonus
+DELETE FROM `creature_text` WHERE `entry`=19392;
```

Comment:
Need to use `CreatureID` instead of `entry`
```sql
DELETE FROM `creature_text` WHERE `CreatureID`=19392;
```
also please correct the first `creature_text` line `(0,0)` to be the same as in `broadcast_text`
```
WE''RE UNDER ATTACK! RUN FOR COVER!
```
(`broadcast_text` is supposedly used in-game, but still)